client.go

```diff
@@ -38,6 +39,12 @@ type Client struct {
 	didHello   bool     // whether we've said HELO/EHLO/LHLO
 	helloError error    // the error from the hello
 	rcpts      []string // recipients accumulated for the current session
+
+	// Time to wait for command responses (this includes 3xx reply to DATA).
+	CommandTimeout time.Duration
+
+	// Time to wait for responses after final dot.
+	DotTimeout time.Duration
```

Comment:
How does this interact with future `CHUNKING` support?

Reply:
I believe that could be mapped to the timeout for response to `BDAT ... LAST`. Do you think it should have a better name?

Reply:
`SubmissionTimeout` maybe?